releases/v1.2.9.toml

```diff
@@ -0,0 +1,31 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.2.8"
+
+pre_release = false
+
+preface = """\
+The ninth patch release for `containerd` 1.2 provides an update to the gRPC
+vendored codebase to include 3 CVE fixes provided in the upstream v1.23.0 release of
+gRPC. Note that updating gRPC to the current release required small changes to our core
+containerd codebase to match the upstream changes since gRPC v1.12.0. These changes
+have been backported from containerd's master branch, as well as a similar small
+change in ttrpc, requiring that package's vendoring to be updated.
+
+### Notable Updates
+* Cherry-pick update to gRPC 1.23.0. [PR #3586](https://github.com/containerd/containerd/pull/3586) {cherry-picked from changes in master PRs [#3192](https://github.com/containerd/containerd/pull/3192) and [#3581](https://github.com/containerd/containerd/pull/3581)}.
+  - Fixes [grpc/grpc-go#2970](https://github.com/grpc/grpc-go/pull/2970) transport: block reading frames when too many transport control frames are queued.
+  - Addresses CVE-2019-9512 (Ping Flood), CVE-2019-9514 (Reset Flood), and CVE-2019-9515 (Settings Flood).
+  - Other changes can be found in the [gRPC release notes](https://github.com/grpc/grpc-go/releases/tag/v1.23.0).
```

Comment:
* cri: Fix a bug that the default apparmor profile is mistakenly applied to privileged containers with runtime/default specified. (containerd/cri#1239)
* cri: Fix a bug that image can't be pulled if an empty AuthConfig is specified. (containerd/cri#1249)

https://github.com/containerd/containerd/pull/3608
@awesomenix

Reply:
Thanks @Random-Liu! We also have a few other [release/1.2] PRs yet to merge so I will do all as one update to the release notes including the CRI-specific fixes from the re-vendor.